r/src/array_to_vector.cpp

```diff
@@ -628,6 +628,25 @@ class Converter_Int64 : public Converter {
   }
 };
 
+class Converter_Unspecified : public Converter {
```

Comment:
I wonder if this name should be `Converter_Null` since AFAICT the other converters take their name from the Arrow type and not the resulting R type.